src/containers/Etherspot.tsx

```diff
@@ -69,7 +64,7 @@ const Etherspot = ({
   smartWalletOnly,
   onLogout,
 }: EtherspotProps) => (
-  <ThemeProvider theme={merge({}, defaultTheme, themeOverride)}>
+  <ThemeProvider theme={merge({}, darkTheme, themeOverride)}>
```

Comment:
Do we require the dark theme by default?

Reply:
yeah aldin asked for it on my ticket